### DISCUSSION TITLE
Option to show when code was last modified

### DISCUSSION THREAD

**Discussion by wiesehahn** [community/community#14499] [Mobile]
Similar to Github.com it would be nice to see when something was modified for the last time in the App. Maybe optional.

**Comment by iskae** [comment]
Hi @wiesehahn 👋 

Thanks a bunch for your first ever request in GitHub 🎉  We really appreciate it and we hope that you will keep the feedback coming 🙇 

I think it is a great idea 👍  I have created a tracking issue for this. Stay tuned for our future releases 😊

**Reply by Joelsamijon** [reply]
⭐👍

**Reply by wiesehahn** [reply]
Any news on this?

**Reply by wiesehahn** [reply]
It's quite annoying if any github repo is opened in the app automatically and then there is no possibility to see when files were changed. Like this it's not usable for me.
Is there still plans to add something like this?

**Reply by iskae** [reply]
Hi @wiesehahn 👋 

It is still in our backlog, but unfortunately it is not prioritised yet. I will be adding your additional comment in the tracking issue and try to bump up the priority of it 👍  

Sorry for the inconvenience 🙇

**Reply by wiesehahn** [reply]
still same problem, generally I like the app, but its often I want to have a quick overview which files where edited when for the last time. 
In the browser it looks like this
<img src="https://github.com/community/community/assets/41429613/b29d8884-6b28-43c1-9697-3e6a7a500b3d" height="600" />

and in the mobile app like this
<img src="https://github.com/community/community/assets/41429613/e7d42c70-8840-45ec-9c28-3604b1ae8d90" height="600" />

I understand there is not enough space on most devices for the entire info (commit message, date last edited), but at least there would be enough space to add the time last edited.

I would still appreciate this :)
